common.go

```diff
@@ -434,3 +441,20 @@ func GetLanguageByAlias(alias string) (lang string, ok bool) {
 
 	return
 }
+
+// GetLanguagesByGitAttributes returns the language assigned in .gitattributes if the regular expresion
+// matchs with the filename
+func GetLanguagesByGitAttributes(filename string, content []byte, candidates []string) []string {
+	return languageByGitAttribute(filename)
+}
+
+func languageByGitAttribute(filename string) []string {
```

Comment:
Why does this function exists? can its code be part of its caller body?
Shouldn't it be called `languages...` instead of `language...`?